amivapi/tests/events/test_emails.py

```diff
@@ -229,3 +229,55 @@ def test_no_nones_in_emails(self):
         mail = self.app.test_mails[1]
         for field in mail.values():
             self.assertTrue('None' not in field)
+
+    def test_moderator_reply_to(self):
+        # Check whether `reply-to` header is the moderator in email if set.
```

Comment:
I think we should use multiline comments here, because docstring (although I'm not sure weather we use them in this project)

Reply:
Agreed, we definitely should.

Reply:
This was addressed with the latest commits.